std/meta.d

```diff
@@ -954,6 +954,81 @@ unittest
     }
 }
 
+/**
+ * Sorts a AliasSeq using $(D C).
```

Comment:
Nitpick: please use either `$(LREF AliasSeq)` or `$(D AliasSeq)` when referencing other symbols. An alternative would be referring to it in English as "alias sequence".